src/cascadia/TerminalConnection/ConptyConnection.cpp

```diff
@@ -451,6 +486,31 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
         return 0;
     }
 
+    static winrt::event<NewConnectionHandler> _newConnectionHandlers;
+
+    winrt::event_token ConptyConnection::NewConnection(NewConnectionHandler const& handler) { return _newConnectionHandlers.add(handler); };
+    void ConptyConnection::NewConnection(winrt::event_token const& token) { _newConnectionHandlers.remove(token); };
+
+    HRESULT ConptyConnection::NewHandoff(HANDLE in, HANDLE out, HANDLE signal, HANDLE process) noexcept
+    try
+    {
+        auto conn = winrt::make<ConptyConnection>((uint64_t)signal, (uint64_t)in, (uint64_t)out, (uint64_t)process);
```

Comment:
nit: cast style

Reply:
Since we can call the **INTERNAL** constructors for ConptyConnection and don't need to go through the idl, we can simply use HANDLE here. There's no reason for the casting back and forth.

We only have to use uint64 if it is part of the public idl interface.

Reply:
I haven't done enough winrt to know what syntax does that. Enlighten me?

Reply:
You just leave it out of the IDL. winrt::make calls the boring normal C++ constructor for the class in the implementation namespace, so there isn’t anything else to be done. The IDL only specifies how it can be constructed from the outside.

so really it’s just `winrt::make<implementation::ConptyConnection>(literally whatever garbage you want, any C++ types or expressions)`

Reply:
It's apparently already not in the IDL so I changed to calling it with `implementation` and took out the casts. It's compiling now. WILL IT BLEND?

Reply:
IT BLENDS. (Don't breathe this.)